src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/response.with.headers/client/nukleus.rpt

```diff
@@ -24,7 +24,7 @@ property targetInputEst ${nuklei.streams("http", "target")}
 property sourceInputEst ${nuklei.streams("source", "http#target")}
 
 property newTargetOutputId ${nuklei:newStreamId()}
-property newSourceInputEstId ${nuklei:newStreamId()}
+property sourceInputEstId ${nuklei:newStreamId()}
```

Comment:
oh I see I had misunderstood it as being connected with the type of route (e.g. route output new).